src/components/ToolStylePopup/ToolStylePopup.js

```diff
@@ -139,7 +139,9 @@ class ToolStylePopup extends React.PureComponent {
           colorMapKey={colorMapKey}
           style={activeToolStyle}
           isFreeText={isFreeText}
-          hideSlider={hideSlider}
+          hideOpacitySlider={hideSlider}
```

Comment:
Hmm I see that in the `StylePopup` we are injecting props with the same names through redux, but here we are also passing them down. Why are we doing this?

I noticed that for the redaction tool and annotations, the sliders will show up now, which isn't what we want based on the code before.

I think to resolve this issue we can choose to not pass these props down, and inside `getAnnotationStyles.js` and `getToolStyles.js`, we can set the slider properties of redaction annotations or tool to null, so that sliders won't show up for redactions.

Reply:
addressed

Reply:
seems i was not aware, that `hideSlider` was not injected as a prop through redux. that was a bug on my part.